### DISCUSSION TITLE
New lines in prompt dividers?

### DISCUSSION THREAD

**Discussion by AlexanderZhk** [h2oai/h2o-llmstudio#684] [Q&A] [answered]
![image](https://github.com/h2oai/h2o-llmstudio/assets/57565244/ff5bf4b4-d6da-430b-bbc8-0aa180bfbf35)
How to correctly set a newline character here? Do i put "\n" there, or do I just copy a newline character into here?

**Comment by psinger** [comment, marked as answer]
Just add `\n`
